examples/Basic/LI_test.py

```diff
@@ -0,0 +1,5 @@
+from pymeasure.instruments.signalrecovery import DSP7265
+
+if __name__ == '__main__':
+    li = DSP7265("visa://phys8173.campus.tue.nl/GPIB0::11::INSTR")
```

Comment:
I think maybe don't include that test file specific to your setup?

Reply:
Yes. That was a mistake indeed.